helm/_shared/named_templates/_datadog.tpl

```diff
@@ -0,0 +1,16 @@
+{{/*
+  For DataDog REST API
+*/}}
+
+{{- define "vro.datadog.envVars" -}}
+- name: DATADOG_API_KEY
+  valueFrom:
+    secretKeyRef:
+      name: vro-datadog
+      key: DATADOG_API_KEY
+- name: DATADOG_API_KEY_ID
+  valueFrom:
+    secretKeyRef:
+      name: vro-datadog
+      key: DATADOG_API_KEY_ID
+{{- end }}
```

Comment:
Nit: new line

Reply:
no you are totally right to call that.  a fix for that is now in flight running checks.